### PR TITLE
fix Group Admin has no option to Edit a Group Chat

### LINKED ISSUE #20956: Add "Edit name & image"  for group chat in mobile app
# Feature Issue

### User Story

**As an** admin in chat,
**I want to** have the opportunity to edit the chat name and image using an option in the context menu,
**so that** I can easily manage chat settings.

### Notes
This option was hidden in https://github.com/status-im/status-mobile/issues/19882

### LINKED ISSUE #21432: Group Admin has no option to Edit a Group Chat
# Bug Report

## Problem 
[comment]: # (A problem description)

Not sure it is a bug or not implemented but we do need this to be working 

### Reproduction

1. Login to app
2. Go to Messages 
3. Tap on +
4. Create Group Chat
5. Tap on three dots menu on the Group chat screen
6. Choose Group Details option in the bottom sheet menu
7. Tap on three dots menu on Group Details screen

####Design VS Actual behavior 

Image

https://www.figma.com/design/7KIYbhoqNGAIFonE0w9TDz/Messages-for-Mobile?node-id=1-25629&node-type=canvas&m=dev

### Additional Information

- Status version: nightly 14/10
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/21432
fixes https://github.com/status-im/status-mobile/issues/20956

### Summary
Implemented functionality that allows group admins to update group chat details.

**Minor Improvements:**
- Moved all group-related files into a single group directory, keeping create and details organized in one place.
- Made a single group-edit file so we can use the same code for creating and editing groups.
- Updated group screen names to follow a consistent naming style (like screen/group-details).

### Video

https://github.com/user-attachments/assets/79b861a0-3c7d-40c2-977c-286471dff0d5

status: ready

### CODE REVIEW

File: src/status_im/navigation/screens.cljs [hunk 1]
```diff
@@ -172,15 +173,20 @@
 
     {:name      :group-add-manage-members
```
Comment (flexsurfer) [inline src/status_im/navigation/screens.cljs]: :screen/group-add-manage-members, as we anyway making changes here

File: src/status_im/contexts/chat/group/details/view.cljs [hunk 2]
```diff
@@ -123,9 +123,9 @@
   [_]
   [rn/view {:style {:height 8}}])
 
-(defn group-details
+(defn view
```
Comment (flexsurfer) [inline src/status_im/contexts/chat/group/details/view.cljs:126]: 👍

**Review by flexsurfer — APPROVED** [review]

**Comment by Horupa-Olena** [comment]
@Parveshdhull Thanks for you PR. All looks good.

I have only one question - If the admin of a group chat leaves the chat and is later added back, do their admin rights return? Or does the chat continue without an admin?

**Comment by Horupa-Olena** [comment]
Also need close feature request after merge https://github.com/status-im/status-mobile/issues/20956

cc @Parveshdhull

**Comment by Parveshdhull** [comment]
Thank you @Horupa-Olena for testing the PR.

> I have only one question - If the admin of a group chat leaves the chat and is later added back, do their admin rights return? Or does the chat continue without an admin?

I don't know about this. The simplest solution would be to return user admin rights. Or there should be a way to transfer admin rights to another user(will require designs). I am not sure, probably better to discuss this with the team.

**Comment by Parveshdhull** [comment]
> @Parveshdhull Thanks for you PR. All looks good.

Thank you, rebased the PR for merging.

**Pull request merged**